### PR TITLE
+Add Current Date on the record rule test case

### PR DESCRIPTION
**Pull request by wongpratan**
https://github.com/digi-serve/planning/issues/507

## Release Notes

- Add Current Date on the record rule test case

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+export default {
+   /**
+    * helper for scrolling in a grid
+    * @function gridScroll
+    * @param {string} id webix id of the grid
+    * @param {int} h horizontal scroll in pixels
+    * @param {int=0} v veritcal scroll in pixels
+    */
+   gridScroll: async (id, h, v = 0) => {
```
Comment (nh758) [inline utils.js:9]: These would probably better as custom cypress commands so we can reuse them in other tests. (See [`ab_runtime/test/e2e/cypress/support/commands.js`](https://github.com/digi-serve/ab_runtime/blob/master/test/e2e/cypress/support/commands.js)).

File: utils.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+   },
+
+   /**
+    * get ABFactory instance
+    * @function getAB
+    */
+   getAB: async () => {
+      return new Promise((resolve) => {
+         cy.window().then((win) => {
+            resolve(win.AB);
```
Comment (nh758) [inline utils.js:26]: I'm not sure about using `window.AB`. I have the same question for Johnny (see https://github.com/digi-serve/ab_runtime/pull/363#discussion_r1493953078)

**Review by nh758 — CHANGES_REQUESTED** [review]
The test itself looks good, but some thoughts on the utils file.

**Review by guyyoo — APPROVED** [review]
LGTM!

File: utils.js (hunk 2 above)
Comment (Hiro-Nakamura) [reply utils.js:26]: Honestly, I don't like "polluting" the global name space like this.  

However, I can't think of another way to make it available to our tests without doing that, so as long as we keep it to a minimum, I'm thinking of keeping this.
